content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -2810,22 +2932,28 @@ It returns the initial state of the current Agent.
+| Parameter                           | Required | Data type | Notes                                                                                                                                                       |
+| ----------------------------------- | -------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `token`                             | Yes      | `string`  | OAuth token from the Agent's account                                                                                                                        |
+| `timezone`                          | No       | `string`  |                                                                                                                                                             |
+| `reconnect`                         | No       | `bool`    | Reconnecting sets the status to the last known state instead of the default one.                                                                            |
+| `push_notifications`                | No       | `object`  |                                                                                                                                                             |
+| `push_notifications.firebase_token` | No       | `string`  | Firebase device token to allow connecting this instance with the existing push notification instance (to be seen as 1 instance).                            |
+| `push_notifications.platform`       | Yes      | `string`  | OS platform; possible values:`ios`, `android`. Required only when `push_notifications` is included.                                                         |
+| `application`                       | No       | `object`  |                                                                                                                                                             |
+| `application.name`                  | No       | `string`  | Application name                                                                                                                                            |
+| `application.version`               | No       | `string`  | Application version                                                                                                                                         |
+| `away` __*__                        | No       | `bool`    | When `true`, the connection is set to the `away` state. Defaults to `false`.                                                                                |
+| `customer_push_level`               | No       | `string`  | Possible values: `my`, `engaged`, `online` __**__. Defaults to `my` if login creates the first session, otherwise it preserves current customer_push_level. |
```

Comment:
```suggestion
| `customer_push_level`               | No       | `string`  | Possible values: `my`, `engaged`, `online` __**__. Defaults to `my` if [login](#login) creates the first session; otherwise it preserves the current `customer_push_level`. |
```